tasks/main.yml

```diff
@@ -10,8 +10,7 @@
 
 - name: Set up signing keys
   ansible.builtin.include_tasks: "{{ role_path }}/tasks/keys/setup-keys.yml"
-  when: (nginx_enable | bool and nginx_install_from == 'nginx_repository')
-        or nginx_amplify_enable | bool
+  when: (nginx_enable | bool and nginx_install_from == 'nginx_repository') or nginx_amplify_enable | bool
```

Comment:
Can we split this back into two lines? It was split on purpose to help with clarity.